lib/Doctrine/ORM/Tools/SchemaTool.php

```diff
@@ -786,8 +786,8 @@ private function gatherColumnOptions(array $mapping): array
             return [];
         }
 
-        $options                        = array_intersect_key($mappingOptions, array_flip(self::KNOWN_COLUMN_OPTIONS));
-        $options['customSchemaOptions'] = array_diff_key($mappingOptions, $options);
+        $options                    = array_intersect_key($mappingOptions, array_flip(self::KNOWN_COLUMN_OPTIONS));
+        $options['platformOptions'] = array_diff_key($mappingOptions, $options);
```

Comment:
I don't know if I should backport this as well… `Column::getCustomSchemaOptions()` will not return the same thing as previously, not sure how acceptable that is.

Reply:
I'd leave it as is. This API is supported up until 4.0.x, so there's no reason to break anything prior to supporting it.